README.md

```diff
@@ -1 +1,213 @@
+
+## 🛠 Built With Visual Studio Code<a name="built-with"></a>
+HTML, CSS and Linters
+
+<details>
+  <summary>Static Web Page</summary>
+  
+</details>
+
+<details>
+  <summary>Styles</summary>
+ 
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Error Free]**
+- **[User Friendly]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Comming Soon
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Developer can can use linter tool to check for errors, and can identify and fix issues before they become larger problems.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1.Github account
+2.Code Editor
+3.Web Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/sahlminkok/Hello-Microverse.git
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following file:
+Index.html
+
+Open Index.html using live server extension.
+
+### Run tests
+
+To run tests, run the following command:
+
+
+Example command:
+
+```sh
+No tests required 
+```
+
+
+### Deployment
+
+You can deploy this project using:
+github VS Code 
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Sahalu Aminu
+
+- GitHub: [@sahlminkok](https://github.com/sahlminkok)
+- Twitter: [@sahlminkok](https://twitter.com/sahlminkok)
+- LinkedIn: [@sahlminkok](https://www.linkedin.com/in/sahlminkok/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Improve the webpage to be responsive using javascript**
+- [ ] **Add Images and Embed Videos using CSS**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Follow me on github if you like this project.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is copyright of [Sahalu Aminu]
+
+This project is [MIT](LICENSE.md) licensed.
```

Comment:
Currently, when a user clicks on the LICENSE link it results in a 404 ERROR. The reason is that you have used the wrong file path and name. Please consider reviewing it as it has a bad effect on user experience.

## Solution:--

Consider replacing the above line with the following and verify by clicking on it does it opens your LICENSE file

```
This project is [MIT](./LICENSE) licensed.
```

Reply:
@ABDULALI3468 How can I rename the branch without pulling another request